tracer/test/snapshots/AspNetWebApi2Tests.CallSite.Classic.NoFF.__path=_api2_transientfailure_false_statusCode=500.verified.txt

```diff
@@ -2,14 +2,31 @@
   {
     TraceId: Id_1,
     SpanId: Id_2,
+    Name: CustomTracingExceptionHandler.handle-async,
+    Resource: CustomTracingExceptionHandler.handle-async,
+    Service: sample,
+    ParentId: Id_3,
+    Tags: {
+      env: integration_tests,
+      span.kind: server,
+      version: 1.0.0
+    }
+  },
+  {
+    TraceId: Id_1,
+    SpanId: Id_3,
```

Comment:
~Were we expecting an _extra_ span in this unrelated endpoint?~ 🤔 

EDIT: Oh I see, it's because we've added an extra exception handler. I imagine in practice the exception handler would _also_ set the exception in this span too

Reply:
Ah yes, I could do probably set the same exception in the exception handler span. I mostly just wanted to make sure that exception handler was invoked so I just created a minimal span. Hope that implementation is cool with you